Source/SharpDX.Direct3D12/Mapping.xml

```diff
@@ -212,6 +214,11 @@
 
     <map interface="ID3D12Debug" name="DebugInterface"/>
     <remove struct="D3D12_MEMCPY_DEST"/>
+    
+    <!--Remove all fields for D3D12_BUILD_RAYTRACING_ACCELERATION_STRUCTURE_INPUTS as the union with pointers cannot be mapped safely in x86/x64 with C# explicit layout-->
+    <remove field="D3D12_BUILD_RAYTRACING_ACCELERATION_STRUCTURE_INPUTS::.*" />
```

Comment:
What do you mean? Either we should remove the struct entirely and remove methods using it, or provide manual marshaling no?

Reply:
Yup, it will follow next PR where I plan to comple the raytracing stuff, meta object, state object and ray tracing.

Reply:
Ok, so if you don't put it will not compile?